djinn/Makefile

```diff
@@ -5,14 +5,14 @@ CC := gcc
 INCLUDES := -I vendor/glad/include -I vendor/glfw-3.3.8/include -I ./
 
 # Source files
-SOURCES := vendor/glad/src/glad.c djinn.c render.c bin/djinn_example_app.c
+SOURCES := vendor/glad/src/glad.c djinn.c dj_input.c render.c bin/djinn_example_app.c
 
 # Output directory and binary
 OUTDIR := build
 OUTPUT := $(OUTDIR)/djinn_example_app
 
 # Libraries
-LIBS := -lm -lglfw
+LIBS := -lm -ldl -lglfw
```

Comment:
I should note that this will probably only work on Linux right now. it should work on mac with a few small adjustments to link flags and includes but windows will be another challenge